src/sdk/namespace_fs.js

```diff
@@ -778,6 +779,17 @@ class NamespaceFS {
                 read_file = await nb_native().fs.open(fs_account_config, part_path, undefined, get_umasked_mode(config.BASE_MODE_FILE));
                 let read_pos = 0;
                 for (;;) {
+                    if (!interval_body_update) {
```

Comment:
why is it inside the for loop? AFAIU the first if condition is for running it once so why not calling it before the loop and get rid of the first if condition?

Reply:
Because we can fail at the opening (in theory) and it would be an AccessDenied error.
That way I want to have the failure as headers response and not body response since we did not start doing any actual work.